devApps/AdaliOSTestApp/AdaliOSTestApp.csproj

```diff
@@ -138,6 +138,12 @@
       <Project>{aece489d-9ed4-442d-8d8f-0f383aebb781}</Project>
       <Name>Microsoft.IdentityModel.Clients.ActiveDirectory</Name>
     </ProjectReference>
+  </ItemGroup>
+    <PropertyGroup>
+    <CodeAnalysisRuleSet>..\..\ConfigureAwait.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" />
   </ItemGroup>
```

Comment:
note, should be done as a separate task and PR to close on this one.
pls. consider how this is either automatically implemented when creating new projects or how a new project added will be "failing a unit test" if these are not specified